framework/src/main/kotlin/br/com/zup/beagle/widget/layout/Screen.kt

```diff
@@ -21,8 +21,10 @@ import br.com.zup.beagle.core.IdentifierComponent
 import br.com.zup.beagle.core.ServerDrivenComponent
 import br.com.zup.beagle.core.Style
 import br.com.zup.beagle.widget.action.Action
+import br.com.zup.beagle.widget.context.Bind
 import br.com.zup.beagle.widget.context.ContextComponent
 import br.com.zup.beagle.widget.context.ContextData
+import br.com.zup.beagle.widget.context.valueOfNullable
 import br.com.zup.beagle.widget.ui.ImagePath
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/3f9ffc2886d2a2022f20f8c0542e5c494959ee76/framework/src/main/kotlin/br/com/zup/beagle/widget/layout/Screen.kt#L28"></span>The import 'br.com.zup.beagle.widget.ui.ImagePath' is unused.</td>
    </tr>
  </tbody>
</table>